index.html

```diff
@@ -95,7 +100,8 @@ <h3 class="title-subheader">Aug 23-26  //  Portland, OR </h3>
           <a href="#sponsors" class="link white"><li>Sponsors</li></a>
           <a href="#code-of-conduct" class="link white"><li>Code of Conduct</li></a>
           <a href="#faq" class="link white"><li>FAQ's</li></a>
-          <a href="https://ti.to/write-speak-code/2017-conference/" class="register-now link white" target="_blank"><li>Register Now</li></a>
+          <!-- <a href="https://ti.to/write-speak-code/2017-conference/" class="register-now link white" target="_blank"><li>Register Now</li></a> -->
+          <li id="nav-register"><tito-button event="write-speak-code/2017-conference" class="register-now link white">Register Now</tito-button></li>						
         </ul>
```

Comment:
Same here: remove the `id=nav-register` and the classes from the tito button. Adding the specific css in custom:
```css
.nav-section .tito-tickets-button {
	border: none;
	border-radius: 8px;
	color: #EB6851;
	background-color: rgba(0,0,0,0);
	font-size: 12px;
}

@media (min-width: 900px) {
	.nav-section .tito-tickets-button {
		font-size: 14px;
	}
}
@media (min-width: 1165px) {
	.nav-section .tito-tickets-button {
		font-size: 16px;
	}
}
```